ocp-build-data-validator/validator/support.py

```diff
@@ -18,6 +19,10 @@ def is_disabled(parsed):
     return 'mode' in parsed and parsed['mode'] == 'disabled'
 
 
+def is_not_release(parsed)
+    return 'content' in parsed or parsed.get('for_release') == 'false'
```

Comment:
The condition doesn't seem right. Looks like it should be `content' in parsed and not parsed.get('for_release')`

Reply:
updated, used an other way to check